### PR TITLE
Crash fix for transaction send with missing flags 

### LINKED ISSUE #1245: Cli crashes on "flow transactions send test.cdc -n testnet  --authorizer xx"
### Problem

$ flow transactions send test.cdc -n testnet  --authorizer testnetaccount 

❌ Crash detected! 

Use the arrow keys to navigate: ↓ ↑ → ← 
? 🙏 Please report the crash so we can improve the CLI. Do you want to report it?: 
  ▸ Yes, report the crash
    No

### Steps to Reproduce

<share any logs/screenshots or steps to replicate>

### Acceptance Criteria

<if any>

### Context

<what are you currently working on that this is blocking?>

### PR DESCRIPTION
**Pull request by bluesign**
Closes #1245 

## Description

There was a check missing for payer and proposer flags when --signer is not used.

______

For contributor use:

- [X] Targeted PR against `master` branch
- [X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [X] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [X] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: internal/transactions/send.go [hunk 1]
```diff
@@ -95,8 +95,14 @@ func send(
 
 	signerName := sendFlags.Signer
 
-	if signerName == "" && proposer == nil && payer == nil && len(authorizers) == 0 {
-		signerName = state.Config().Emulators.Default().ServiceAccount
+	if signerName == "" {
+		if proposer == nil && payer == nil && len(authorizers) == 0 {
+			signerName = state.Config().Emulators.Default().ServiceAccount
```
Comment (bjartek) [inline internal/transactions/send.go:100]: Will this not be wrong and give a bad error message if network is not emulator?

**Review by bjartek — CHANGES_REQUESTED** [review]

File: internal/transactions/send.go (hunk 1 above)
Comment (bluesign) [reply internal/transactions/send.go:100]: it is current behaviour, we can change it ofc but probably people rely on that.

**Comment by bluesign** [comment]
I have some CI errors that doesn't make sense, anyone has any idea?

File: internal/transactions/send.go (hunk 1 above)
Comment (bjartek) [reply internal/transactions/send.go:100]: True. The diff fooled me.

**Review by bjartek — APPROVED** [review]

**Review by chasefleming — APPROVED** [review]

**Pull request merged**